backend/.env.sample

```
@@ -7,4 +7,6 @@ JWT_REFRESH_SECRET_KEY=
 REDIS_PASSWORD=
 REDIS_HOST=
 REDIS_PORT=
-FRONTEND_URL=
\ No newline at end of file
+GITHUB_WEBHOOK_SECRET=
+EMAIL_USER=  
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)